lib/units/validateBody.js

```diff
@@ -69,7 +69,11 @@ function getBodyType(body, contentType, httpMessageOrigin) {
   const hasJsonContentType = isJsonContentType(contentType);
 
   try {
+    // Use strict JSON parsing for actual body.
+    // This prevents any kind of normalization of trailing commas
+    // and other custom transformations applied by "jju".
```

Comment:
We removed jju, so this comment is extraneous now.

Reply:
Removed in 3aade08.